src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1574,9 +1574,8 @@ impl Coordinator {
         })?;
 
         let json_string = json_string(&json_value);
-        Ok(Self::send_immediate_rows(vec![Row::pack_slice(&[
-            Datum::String(&json_string),
-        ])]))
+        let row = Row::pack_slice(&[Datum::String(&json_string)]);
+        Ok(Self::send_immediate_rows(row))
```

Comment:
Possibly untested (if coverage data is accurate).

Reply:
looks like this is called when running `EXPLAIN SCHEMA ...` which I'm not surprised our existing tests don't cover. It's not a major part of this PR though so I'm going to skip if that's okay with you @nrainer-materialize